sources/getRole.ts

```diff
@@ -83,23 +135,26 @@ const prohibitedAttributes: Record<string, Set<string>> = {
  * @param role The role used for this element. This is specified to control whether you want to use the implicit or explicit role.
  */
 function hasGlobalAriaAttributes(element: Element, role: string): boolean {
-	// https://rawgit.com/w3c/aria/stable/#global_states
```

Comment:
Aria we should continue to track stable until we replaced aria-query in `@testing-library/dom` so that we don't diverge.

Reply:
Curiously stable appears to be an alias unique to the WAI-ARIA spec which is a synonym for the master branch as far as I can tell (https://rawgit.com/w3c/aria/master/), meaning it is tracking the WAI-ARIA 1.3 Editor's Draft currently (almost, but not quite - it's actually more bleeding edge than the Editor's Draft).

I've raised https://github.com/A11yance/aria-query/issues/498 to get clarification as the README for aria-query implies it's tracking WAI-ARIA 1.2 which conflicts with where the link goes (and _I think_, what has been implemented).

In the meantime will update using:

- https://w3c.github.io/accname/ - ACCNAME 1.2 W3C Editor's Draft
- https://rawgit.com/w3c/aria/stable/ - WAI-ARIA 1.3 W3C (Pre) Editor's Draft
- https://www.w3.org/TR/html-aam-1.0/ - HTML-AAM 1.0 W3C Working Draft
- https://www.w3.org/TR/html-aria/ - HTML-ARIA W3C Recommendation

Where precedent has been set by the two repos incl. https://github.com/A11yance/aria-query/pull/447.